images/Dockerfile.test

```diff
@@ -0,0 +1,14 @@
+from golang:1.11-alpine3.8 as runner
+
+RUN apk add --update nodejs nodejs-npm
+RUN apk add git gcc libc-dev python make g++
+RUN go get github.com/golang/protobuf/proto
```

Comment:
we need to fix this and avoid pulling in whatever random version of proto is available at build time.

Reply:
`dep ensure` doesn't want to add this, so I'm still confused why we're adding it.

Reply:
take it out and create the image. you will see an error related to ginkgo depending on it. I didn't spend too much time investigating the issue. Doing a `go get` seemed to fix it so I left it there. we should do a more thorough investigation later on. For now lets pull that in and configure Travis with the image so that we can run all the tests for bind. We can come back to this and optimize it.

Reply:
did not fail for me, at least not obviously.

Reply:
Was this resolved?

Reply:
I didn't have an issue either when i deleted the proto line. What version of ginkgo are you using? Maybe we should lock down the version of ginkgo so we are all consistent.